testing/cage/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=cage
+pkgver=0.1.1
+pkgrel=0
+pkgdesc="A Wayland Kiosk"
+url="https://github.com/Hjdskes/cage"
+arch="all"
+license="MIT"
+depends="xorg-server-xwayland"
+makedepends="meson cmake wlroots-dev wayland-protocols"
+options="!check" #No checks
+source="$pkgname"-"$pkgver".tar.gz::https://github.com/Hjdskes/cage/archive/v"$pkgver".tar.gz
+
+prepare () {
+	meson --buildtype=release -Dxwayland=true --prefix /usr "$srcdir/build"
+}
+	
+build () {
+	ninja -C "$srcdir/build"
+}      
```

Comment:
>IC:[AL8]:APKBUILD:21:trailing whitespace
TC:[AL10]:APKBUILD:15:do not use space before function parenthesis
TC:[AL10]:APKBUILD:19:do not use space before function parenthesis

Couple of linting issues.

Reply:
I think these should all be taken care of now